std/file.d

```diff
@@ -4308,7 +4307,17 @@ string tempDir() @trusted
         }
         else static assert(false, "Unsupported platform");
 
-        if (cache is null) cache = getcwd();
+        if (cache is null)
+        {
+            import std.path : dirSeparator;
+            cache = getcwd() ~ dirSeparator;
+        }
     }
     return cache;
 }
+
+@trusted unittest
```

Comment:
Why is this unittest marked `@trusted`?  As far as I know, it should be either `@safe` or `@system`. If necessary, split off the `@system` parts into a separate unittest block so that the rest can be marked `@safe`.